static/app/components/projectPageFilter.tsx

```diff
@@ -139,13 +144,13 @@ export function ProjectPageFilter({
     ) : (
       <IconProject />
     );
+
     return (
       <PageFilterDropdownButton
         detached
         hideBottomBorder={false}
         isOpen={isOpen}
         highlighted={desyncedFilters.has('projects')}
-        {...getActorProps()}
```

Comment:
The reason we don't need this is bc the `ProjectSelector` is already in a `DropdownAutoComplete` which is wrapped in an actor.